src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -399,6 +367,72 @@ export default class WorkloadPodLogs extends React.Component<WorkloadPodLogsProp
     );
   }
 
+  private getSplitter = () => {
```

Comment:
I like seeing this broken up as it is cleaner as it was getting big.

Reply:
It also helped set up to use different props for the different orientations.